install.sh

```
@@ -0,0 +1,196 @@
+  printf "${BOLD}game-ci installed successfully!${RESET}\n"
+  printf "\n"
+
+  # Check if install dir is in PATH
+  case ":$PATH:" in
+    *":${INSTALL_DIR}:"*)
+      info "game-ci is already in your PATH. Run: game-ci --help"
+      ;;
+    *)
+      SHELL_NAME=$(basename "$SHELL" 2>/dev/null || echo "sh")
+      case "$SHELL_NAME" in
+        zsh)  PROFILE="~/.zshrc" ;;
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)